app/views/about/terms.ja.html.haml

```diff
@@ -0,0 +1,76 @@
+- content_for :page_title do
+  #{site_hostname} Terms of Service and Privacy Policy
+
+.wrapper
+  %h2 プライバシーポリシー
+
+  %h3#collect どのような情報を収集するのですか？
+
+  %p あなたがこのサイトに登録すると、ここで共有された情報を読んだり、書いたり、評価したりして、フォーラムでの情報を集める事ができます。
+
+  %p このサイトに登録する際には、名前とメールアドレスの入力を求めることがあります。ただし、登録をすることなくこのサイトを利用することも可能です。あなたのメールアドレスは、固有のリンクを含んだメールで確認されます。そのリンクにアクセスした場合にメールアドレスを制御することとなります。
+
+  %p アカウントを登録し、投稿を行った際にはその投稿が行われたIPアドレスを記録します。また、このサーバーに対する全てのリクエストはIPアドレスを含むサーバーログとして保管されます。
+
+  %h3#use 自分の情報を何に使うのですか？
+
+  %p このサイトで収集された情報は、次のいくつかの方法で使用されます:
+
+  %ul
+    %li パーソナライズ・エクスペリエンス &mdash; あなたの情報は、あなたや他のユーザーのニーズに対応するために役立ちます。
+    %li サイトの改善・最適化 &mdash; このサービスはあなたから受け取った情報やフィードバックに基づいて提供されるサイトの改善を行いつづけます。
+    %li サービスの向上 &mdash; あなたの情報は、ユーザーからの要求やサポートへより効果的に対応するために役立ちます。
+    %li 定期メールの送信 &mdash; メールアドレスは、情報の送信、トピックの変更やユーザー名に関係するお知らせ、お問い合わせに関する返答、その他のリクエストや質問に関してお知らせするために使用されます。
+
+  %h3#protect 自分の情報はどのように保護されるのですか？
+
+  %p このサービスはあなたの個人情報の入力、送信、またはアクセスに際してあなたの個人情報の安全性を維持するために様々なセキュリティ手段をとっています。
+
+  %h3#data-retention どのようにデータは保存されますか？
+
+  %p このサービスは次のことを行います:
+
+  %ul
+    %li このサーバーへのすべての要求に対して、IPアドレスを含むサーバーログを90日以内に渡って保持します。
+    %li 登録されたユーザーとその投稿に関連付けされたIPアドレスを5年以内に渡って保持します。
+
+  %h3#cookies クッキーを使用していますか？
+
+  %p はい。クッキーはあなたがウェブブラウザ上で許可した場合にコンピュータのストレージに転送される小さなファイルです。これらのクッキーを使用すると、サイトでブラウザが識別され、登録済みのアカウントを持っている場合は登録済みのアカウントに関連付けがされます。
+
+  %p クッキーを使用して、今後再度閲覧された場合に前回のデータから設定を呼び出したり、今後の改善のためにサイトのトラフィックやサイトの相互作用に関する集計データを作成します。このサービスは、サイトを訪れた方との理解を深めるために、第三者のサービス提供者と契約することがあります。これらのサービス提供者というものは、このサービスでの業務を行ったり、改善するためにこのサービスの代わって収集された情報を使用することはできません。
+
+  %h3#disclose このサイトは外部に何らかの情報を開示していますか？
+
+  %p 私たちは、個人を特定出来る情報を外部へ販売、取引、または他の方法で渡すことはありません。これには、このサイトを操作したり、業務を行ったり、サービスを提供するのに役立つ信頼できる第三者は含まれません。法令遵守、サイトポリシーの施行、このサービスや他の人の権利、財産または安全の保護のために適切であると判断した場合に、あなたの情報を公開する場合があります。ただし、マーケティングや広告、その他の目的で匿名での訪問者情報を他者へ提供することができます。
+
+  %h3#third-party サードパーティのリンク
+
+  %p ときどき、このサービスの方針にもとづいてこのサイトや第三者のサービスを提供することがあります。これらの第三者のサイトには、個別の独立したプライバシーポリシーがあります。従って、これらのリンク先のサイトに関するコンテンツや活動にかんしては一切責任を負いません。ですが、サイトの完全性やこれらのサイトに関するフィードバックは非常に重要なものであると認識しております。
+
+  %h3#coppa 子供のオンライン・プライバシー保護法
+
+  %p
+    このサイト、製品、サービスはすべて13歳以上の人を対象としております。このサーバーが米国にあり、13歳未満の場合はCOPPA
+    = surround '(', '),' do
+      = link_to 'Children\'s Online Privacy Protection Act', 'https://en.wikipedia.org/wiki/Children%27s_Online_Privacy_Protection_Act'
+    にもとづいてこのサイトを使用しないでください。
+
+  %h3#online オンライン限定のプライバシーポリシー
+
+  %p このオンライン・プライバシーポリシーは、このサイトを通じて収集された情報のみに適用され、オフラインで収集される情報には適用されません。
+
+  %h3#consent あなたの同意
+
+  %p このサービスを使用することにより、このサイトのプライバシーポリシーに同意するものとします。
+
+  %h3#changes プライバシーポリシーの変更
+
+  %p ライバシーポリシーを変更する場合は、このページへ変更内容を掲載します。
+
+  %p この文章のライセンスはCC-BY-SAです。このページは2013年5月31日が最終更新です。
```

Comment:
I suggest to update the date, as the Norwegian version does.
https://github.com/tootsuite/mastodon/commit/e1264bbd92718cd6d78bed8c2d1ad9cb96384fea#diff-f38cf3cfcef9d50514d9021e86477a4cR71